windows/entrypoint.sh

```diff
@@ -6,6 +6,7 @@ BRANCH=$1
 JOBS=$2
 DEBUG=$3
 REVISION=$4
+DEBUG_SYMBOLS_PATH=$5
```

Comment:
There's no need to use another parameter.

Reply:
I might be missing something but if this improves readability, why not add it?